index.html

```diff
@@ -62,6 +62,157 @@ <h1>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="" alt="">
+                </div>
+                <div class="card_content">
+                    <h1>Multi-Post Stories <br> Gain+Glory</h1>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="" alt="">
+                </div>
+                <div class="card_content">
+                    <h1>Multi-Post Stories <br> Gain+Glory</h1>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="" alt="">
+                </div>
+                <div class="card_content">
+                    <h1>Multi-Post Stories <br> Gain+Glory</h1>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- About -->
+    <section class="about">
+        <div class="about_content">
+            <div>
+                <h1>About Me</h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+                <button>Get my Resumes</button>
+            </div>
+            <div class="about_boxes">
+                <div class="box">
+                    <div> <img src="Assets/icon-languages.png" alt=""></div>
+                    <h1>Languages</h1>
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="box">
+                    <div> <img src="Assets/icon-frameworks.png" alt=""></div>
+                    <h1>Frameworks</h1>
```

Comment:
it is important to include alternative text (alt text) for your images to ensure accessibility for all users and it would be better to use meaningful classes so your code would be more readable🥱🥱